pkg/chunked/cache_linux.go

```diff
@@ -398,7 +398,8 @@ func prepareMetadata(manifest []byte) ([]*internal.FileMetadata, error) {
 	toc, err := unmarshalToc(manifest)
 	if err != nil {
 		// ignore errors here.  They might be caused by a different manifest format.
-		return nil, nil
+		logrus.Debugf("could not unmasshal manifest: %v", err)
```

Comment:
Typo: "unmasshal" should be "unmarshal".

Reply:
Fixed